test/unit/edgeware/enricher.spec.ts

```diff
@@ -774,7 +785,24 @@ describe('Edgeware Event Enricher Filter Tests', () => {
       }
     });
   });
+  /** offences events */
+  it.only('should enrich new offence event', async () => {
+    const kind = EventKind.Offence;
+    const event = constructEvent([ 'offline', '10000', constructBool(true), offenceDetails ],
+      'offences', [ 'Kind', 'OpaqueTimeSlot', 'boolean', 'Option<OffenceDetails>[]' ]);
```

Comment:
You probably don't need the 2nd and 3rd arguments here, but if you wanted to keep them then I'd make sure you replace `boolean` with `bool` to ensure it uses the correct Codec type.